test/Compatibility/accessibility.swift

```diff
@@ -187,10 +187,10 @@ public class Base {
 
 
 public extension Base {
-  open func extMemberPublic() {} // expected-warning {{'open' modifier conflicts with extension's default access of 'public'}} {{none}}
+  open func extMemberPublic() {} // expected-warning {{non-@objc methods in extensions cannot be overriden; use 'public' instead}} {{3-7=public}}  // expected-warning {{'open' modifier conflicts with extension's default access of 'public'}} {{none}}
 }
 internal extension Base {
-  open func extMemberInternal() {} // expected-warning {{'open' modifier conflicts with extension's default access of 'internal'}} {{none}}
+  open func extMemberInternal() {} // expected-warning {{non-@objc methods in extensions cannot be overriden; use 'public' instead}} {{3-7=public}} // expected-warning {{'open' modifier conflicts with extension's default access of 'internal'}} {{none}}
```

Comment:
That just make me wonder if we should say `use '${parent_decl_modifier}'` in this case `use 'internal'` instead, or even if we should emit this warning if the extension decl is less than 'public' ... @varungandhi-apple?

Reply:
That's a good point. In this case, I think we should skip the `non-@objc` warning because the other warning is more accurate.

Reply:
Makes sense! 👍 I fixed the code and test to skip this warning in such cases.